py/htm/examples/scalar_encoder.py

```diff
@@ -95,7 +95,9 @@
     else:
         n_samples = (enc.parameters.maximum - enc.parameters.minimum) / enc.parameters.resolution
         oversample = 2 # Use more samples than needed to avoid aliasing & artifacts.
-        n_samples  = int(round( oversample * n_samples ))
+        n_samples  = int(round( oversample * n_samples )) #TODO should this be done already in the c++?
```

Comment:
do we have the concept of "category encoder" in c++? If so, the modification should be moved there

Reply:
`n_samples` controls the number of inputs which it encodes to make the graphic.  It determines the which inputs are given to the encoder.